samples/addBucketLabel.js

```diff
@@ -41,12 +41,13 @@ function main(
   const labels = {
     [labelKey]: labelValue,
   };
+
   async function addBucketLabel() {
     await storage.bucket(bucketName).setLabels(labels);
-    console.log(`Added label to bucket ${bucketName}.`);
+    console.log(`Added label to bucket ${bucketName}`);
   }
 
-  addBucketLabel();
+  addBucketLabel().catch(console.error);
```

Comment:
👋   we don't want these `.catch(console.error)` things spread about.  This isn't how customers would handle errors in production, so we want to let it throw.  Adding in @fhinkel who owns the node.js rubric to chime in as well.

Reply:
Haha, whoops. It seemed like more samples had them than not, so I caught up the ones that didn't. I'll wait for @fhinkel confirmation, but happy to revert and remove any others.